src/components/implementation/nicmgr/dpdk/interface_impl.c

```diff
@@ -0,0 +1,203 @@
+#include <cos_types.h>
```

Comment:
I don't like the name of this file. iterface_impl makes very less sense to the user. Maybe rename it as nic_mgr_coordination.c or something like that.